src/components/Table/TableBody.tsx

```diff
@@ -48,7 +54,7 @@ export interface ITableBodyProps {
  * - Renders row out of order indicator
  * - Renders next page loading UI (`RowsSkeleton`)
  */
-export const TableBody = memo(function TableBody({
```

Comment:
I had to remove memo because with it the component wasn't updating on row selection. I think it is because on row selection the `table` prop changes but memo does a shallow rendering and thus doesn't re-render the component.